package-scripts.cjs

```
@@ -175,10 +188,18 @@ module.exports = {
-      teardown: series.nps('proxy.stop', 'gitserver.stop'),
+      setup:
+        process.platform === 'win32'
+          ? series.nps('gitserver.start')
+          : series.nps('proxy.start', 'gitserver.start'),
+      teardown:
+        process.platform === 'win32'
+          ? series.nps('gitserver.stop')
+          : series.nps('proxy.stop', 'gitserver.stop'),
       node: process.env.CI
-        ? `export ${jestEnv}\n${retry3(timeout15(jestCommand))}`
+        ? (process.platform === 'win32'
```

ESLint

[error] Delete `(`

(prettier/prettier)